src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -437,14 +440,18 @@ class SchedulerActions(
     scale(driver, app)
   }
 
-  def stopApp(driver: SchedulerDriver, app: AppDefinition): Future[_] = {
+  def stopApp(app: AppDefinition): Future[_] = {
     healthCheckManager.removeAllFor(app.id)
 
     log.info(s"Stopping app ${app.id}")
     taskTracker.appTasks(app.id).map { tasks =>
+      tasks.foreach { task =>
+        if (task.launchedMesosId.isDefined) {
+          log.warn("Killing {}", task.taskId)
```

Comment:
Yes, the `TaskKillServiceDelegate` now logs kill requests and their respective reasons. changed the others to debug/info